feat: support enabling the continuous profiler (#358)
DataDog/serverless-plugin-datadog: README.md

| `coldStartTraceMinDuration`   | Sets the minimum duration (in milliseconds) for a module load event to be traced via Cold Start Tracing. Number. Defaults to `3`. |
| `coldStartTraceSkipLibs`      | optionally skip creating Cold Start Spans for a comma-separated list of libraries. Useful to limit depth or skip known libraries. Default depends on runtime. |
| `subdomain`                   | Set an optional subdomain to use for app URLs which are printed to output. Defaults to `app`. |

> | `enableProfiling`             | Enable the Datadog Continuous Profiler with `true`. Supported in Beta for NodeJS and Python. Disabled by default. |

Reviewer: Recommend using the same format as other fields.
```suggestion
| `enableProfiling`             | Enable the Datadog Continuous Profiler with `true`. Supported in Beta for NodeJS and Python. Defaults to `false`. |
```
